src/app/modules/main/pages/main/main.component.spec.ts

```diff
@@ -20,4 +20,9 @@ describe('MainComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should be false', () =>{
```

Comment:
Try to improve the name of the tests.

E.g., _it should set the status as false while viewport gets thinner._

Reply:
Done!